front-end/src/pages/HomePage.tsx

```diff
@@ -32,9 +38,12 @@ export default function HomePage(props:any) {
     setUserSearch(e.target.value);
   }
 
+  
+  //search for authors with the same displayName
   function searchUsers(e:any){
     e.preventDefault();
-    props.history.push('/authors/'+userSearch);
+    const URL = process.env.REACT_APP_API_URL + "/api/authors/";
+    ReactDOM.render(<AuthorList url = {{searchURL: URL}}/>, document.getElementById("belowRoot")); 
```

Comment:
Not sure if we should be doing this and I don't know what would be better to do - maybe someone else can help. I think we should be navigating to the `AuthorPage`?

Reply:
This url is used to get a list of authors with the same display name. The AuthorPage seems to only be accessible by a loggedInUser.

Reply:
I think we should create a new page for the `authorList` component, and call it `AuthorSearchPage` or something similar. We can then render what you have here in that new page. With this new feature, we should be able to access the profile page without a loggedInUser.

Hopefully we already added logic to the `AuthorPage` if the `loggedInUser` is undefined

Reply:
I don't think we've added that logic yet. I think for Friday it might be a good idea to just have search results displayed on a new page. Right now, we should be able to see a GET request to Django and then get an unauthorized response when trying to access another Author's page that isn't your own or if you aren't logged in. I don't think this response shows up on the frontend but it will on the Django console. Might need to double check this